app/plugins/prometheus.md

```diff
@@ -36,6 +36,11 @@ specifying `admin_listen = off`) will need to use a [custom Nginx
 configuration template](/latest/configuration/#custom-nginx-configuration)
 to expose the metrics data.
 
+#### Grafana dashboard
```

Comment:
Let's add the section to the right-sided navbar (see yaml definitions in the meta section at the top of this page)

Reply:
Re-uped the commit.

Reply:
@thibaultcha Good point.
I propose:
- Reading metrics
  - Grafana Dashboard
  - Available metrics
  - Accessing the metrics

Thoughts?

Reply:
ping @thibaultcha

Reply:
Sounds weird to me, not sure why we need a top-level title if all other subtitles belong to it; nor even if all subtitles are properly scoped under a "reading metrics" title - but I have no strong feelings either way.

Reply:
@thibaultcha I see your point. I've updated to all being `<h4>` and removed "Reading metrics" since that was not appropriate.

Reply:
Remember to remove "reading metrics" from the sidebar (i.e. from the YAML on top) as well.

Reply:
Oops, thanks for catching that! Re-uped.

Reply:
@thibaultcha ping. Apologies for the back and forth on this.